### ISSUE TITLE
Final production checks

### ISSUE THREAD

**Issue by madskristensen** [madskristensen/HomeAutomationCookbook#13]
Before announcing this new site to home automaters worldwide, double check the entire site for violations of best practices, accessibility, SEO, writing style and content consistency, and any other issues that should be addressed to be considered production ready
